README.md

```diff
@@ -41,76 +34,77 @@ The current set of features include:
 
-We need to split the build process between Ament and the rest of `ros2_java` workspace so that the additional build type for Gradle projects is picked up by Ament.
+        sudo apt install default-jdk
 
-Make sure you have Gradle 3.2 (or later) installed. Ubuntu 16.04 ships with Gradle 2.10, you can install a more recent version of Gradle from the following PPA:
+1. Install Gradle.
+Make sure you have Gradle 3.2 (or later) installed.
 
-```
-$ sudo add-apt-repository ppa:cwchien/gradle
-$ sudo apt install -y gradle
-```
-For OSX, use homebrew command to install gradle:
+    *Ubuntu Bionic or later*
 
-```
-$ brew install gradle
-```
+        sudo apt install gradle
 
-For OSX users, there some compatibily issues have been reported between gradle 3.x and Java 9. Currently only the combination of Gradle 3.x + Java 8 has been tested successfully. Make sure to use install and use Java 8 for the building process:
+    *macOS*
 
-```
-$ brew tap caskroom/versions
-$ brew cask install java8
-$ export JAVA_HOME=/Library/Java/JavaVirtualMachines/1.8.0.jdk/Contents/Home
-``` 
+        brew install gradle
 
-> Note: On Windows, you may use `choco install -y gradle`
+    Note: if run into compatibily issues between gradle 3.x and Java 9, try using Java 8,
 
-The following sections deal with building the `ros2_java` codebase for the desktop Java runtime and for Android.
+        brew tap caskroom/versions
+        brew cask install java8
+        export JAVA_HOME=/Library/Java/JavaVirtualMachines/1.8.0.jdk/Contents/Home
 
-Desktop
--------
+    *Windows*
 
-```
-mkdir -p ros2_java_ws/src
-cd ros2_java_ws
-curl -skL https://raw.githubusercontent.com/ros2-java/ros2_java/dashing/ros2_java_desktop.repos -o ros2_java_desktop.repos
-vcs import src < ros2_java_desktop.repos
-. ../ament_ws/install_isolated/local_setup.sh
-ament build --symlink-install --isolated
-```
+        choco install gradle
+
+1. Install build tools:
+
+        sudo apt install curl python3-colcon-common-extensions python3-pip python3-vctool
+
+1. Install Gradle extensions for colcon:
+
+        python3 -m pip install -U git+https://github.com/colcon/colcon-gradle
+        python3 -m pip install --no-deps -U git+https://github.com/colcon/colcon-ros-gradle
+
+### Download and Build ROS 2 Java for Desktop
+
+1. Source your ROS 2 installation, for example:
+
+        source /opt/ros/dashing/setup.bash
+
+1. Download the ROS 2 Java repositories into a workspace:
+
+        mkdir -p ros2_java_ws/src
+        cd ros2_java_ws
+        curl -skL https://raw.githubusercontent.com/ros2-java/ros2_java/dashing/ros2_java_desktop.repos | vcs import src
+
+1. **Linux only** Install ROS dependencies:
+
+        rosdep install --from-paths src -y -i --skip-keys "ament_tools"
+
+1. Build desktop packages:
 
-> On Windows, if you would like to use OpenSplice, call `call "C:\opensplice67\HDE\x86_64.win64\release.bat"` before building.
+        colcon build --symlink-install
```

Comment:
IIRC `--symlink-install` won't work on Windows, this needs a note about that.

Reply:
Good call. Added note for Windows to use `--merge-install`: 193bd7d